README.md

```diff
@@ -1 +1,53 @@
-# Metrics-Webapp
\ No newline at end of file
+# Metrics-Webapp
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Additional description about the project and its features.
+
+This project was built to fetch data from an api an map out specific companies location and their prices.
+
+![](/src/assets/navigation.PNG)
+![](/src/assets/Page1.PNG)
+![](/src/assets/Page2Style.PNG)
+![](/src/assets/test.PNG)
+
+
+# Built With
+
+1. React and Redux
+2. Linters (Stylelint, Lighthouse)
+
+# Getting Started
+
+## Installation
+1. You can run: `npm install create-react-app` (to create a new react app) 
+2. After the installation of the react app you can run: `npm start` (to load it on the server)
+3. Before you run test you first have to install `jest`in order to run test you can as well add `--coverage`to the script array in your package.json
+4. Then after the previewing in locally you can now run : `npm test` (to see if the code works perfectly.
+
+To get a local copy up and running follow these simple example steps.
+
+# Prerequisites
+
+Setup: Clone https://github.com/oluyaratosin123/Metrics-Webapp.git into the destination folder of your local machine.
+Install: Install linters with the npx commands given in the linters.yml file.
+Usage: N/A
+Run tests: Run syntax tests with the use of the npx commands given in the linters.yml file.
+Deployment: Run index.html on your local server.
+
+# Authors
+
+GitHub: https://github.com/oluyaratosin123
+LinkedIn: https://www.linkedin.com/in/edward-oluyara-708b88215/
+
+Author name: Nelcon Sakwa 
+Behance: https://www.behance.net/sakwadesignstudio
```

Comment:
- [OPTIONAL] You referencing the design's author is a good practice. However, I want to suggest that since you are acknowledging the author, it will be nice if you could place it under the acknowledgement title.

See example:

### Acknowledgments 
- [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).